src/content/hinting.ts

```diff
@@ -844,6 +844,14 @@ export function hintables(selectors = DOM.HINTTAGS_selectors, withjs = false) {
         const elemSet = new Set([...elems, ...DOM.hintworthy_js_elems])
         elems = [...elemSet]
     }
+    elems.forEach((element, index) => {
```

Comment:
Could you add a note (either in a comment or a commit message) which issue this fixes?

Just from looking at the code alone it isn't obvious why it is needed (which is probably why the code wasn't here initially and we had the issue :) )